src/components/LoanCards/Buttons/ActionButton.vue

```diff
@@ -88,14 +108,76 @@ export default {
 
 			return this.isSimpleLendButton ? Lend25Button : LendIncrementButton;
 		},
+		canParticipateInExperiment() {
+			// Determines if a user is eligible to be part of the experiment.
+			// - If they added first loan to basket
+			// - If they do not have promo credits (this is handled below in the add to basket event)
+			// - If they have not clicked on "continue browsing" on the checkout page
+			if (this.itemsInBasket.length === 1 && !this.userPrefContinueBrowsing) {
+				return true;
+			}
+			return false;
+		}
+	},
+	mounted() {
+		this.userPrefContinueBrowsing = store2('userPrefContinueBrowsing') === true; // read from localstorage
 	},
 	methods: {
 		handleAddToBasketEvent(payload) {
 			this.$emit('add-to-basket', payload);
+
 			if (payload.success) {
-				this.triggerAddToBasketInterstitial(payload.loanId);
+				if (this.canParticipateInExperiment) {
+					// If user can participate, make query for hasFreeCredits
+					// By gating this, it avoids unnecessarily calling this query.
+					this.apollo.query({
+						query: freeCreditQuery,
+					}).then(data => {
+						this.hasFreeCredits = _get(data, 'shop.basket.hasFreeCredits');
+
+						// Users with free credits go to legacy basket.
+						if (!this.hasFreeCredits) {
+							// User is eligible for experiment, track event
+							this.$kvTrackEvent(
+								'Lending',
+								'EXP-GROW-127-Jul2020',
+								this.addToBasketRedirectExperimentShown === true ? 'b' : 'a'
+							);
+
+							if (this.addToBasketRedirectExperimentShown) {
+								// User is part of shown group
+								this.$router.push({
+									path: '/basket',
+								});
+							} else {
+								// User is part of control group
+								this.triggerAddToBasketInterstitial(payload.loanId);
```

Comment:
This repetition of `triggerAddToBasketInterstitial` is ugly, but I was having trouble finding a better way to rework this.